modules/govuk_jenkins/templates/jobs/clear_cdn_cache.yaml.erb

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: clear-cdn-cache
+    display-name: Clear production CDN cache
+    project-type: freestyle
+    description: "Clears the production CDN cache for the 10 most popular pages on GOV.UK."
```

Comment:
Likely true, but I'm inclined to leave it because I've found YAML string handling to be finicky and the description is the one most likely to get special characters added to it later.

What do you think?

Reply:
I don't really mind, I just prefer to not have the quotes where it's not required. It might be better do to the following which should avoid any issues with special characters:

```yaml
description: >
  Clears the production CDN cache for the 10 most popular pages on GOV.UK.
```

Reply:
Sounds fine.